feature/gnmi/ate_tests/telemetry_basic_check_test/telemetry_basic_check_test.go

```diff
@@ -0,0 +1,884 @@
+/*
```

Comment:
Convert to line comments? Just remove this block and re-add the license using addlicense.